politeiawww/www.go

```diff
@@ -619,6 +619,7 @@ func _main() error {
 		p.checkInvoiceNotifications()
 
 		p.setupCMSAddressWatcher()
```

Comment:
This needs to be done in a separate go routine.  If dcrdata wasn't able to be connected to on initialization the `Reconnect` call at the beginning of `setupCMSAdressWatcher` will be blocking.